test/martha/forceAccessUrl.test.js

```diff
@@ -0,0 +1,68 @@
+const test = require('ava');
+const sinon = require('sinon');
+const config = require('../../common/config');
+const helpers = require('./_helpers.js')
+
+const { marthaV3Handler: marthaV3 } = require('../../martha/martha_v3');
+const apiAdapter = require('../../common/api_adapter');
+
+const terraAuth = "Bearer XYZ_TerraAuthToken";
+const bucketId = "bucket_id";
+const responseId = `v1_ResponseId_${bucketId}`;
+const gsAccessId = "gcp-us-central1";
+const gsUrl = `gs://broad-jade-dev-data-bucket/BucketIdPrefix/${bucketId}`;
+const drsUri = `drs://${config.HOST_TDR_DEV}/${responseId}`
```

Comment:
`config.HOST_TDR_DEV` appears to be required for this test to pass. Is this really a unit test after all?